### PR TITLE
Saurav/nightly updates

### PR DESCRIPTION
**Pull request by AirSurfer09**
generate Audio config added

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 node_modules
 /src
 /_proto
+CharacterUtil.js
```
Comment (crazylazylife) [inline .gitignore]: Can you make it a part of the src folder.

**Review by crazylazylife — COMMENTED** [review]
We do not need the "Proto/service" folder outside "dist". Can you remove that. I do not think it was there originally.

File: Proto/service/service.proto [hunk 2]
```diff
@@ -87,13 +95,16 @@ message GetResponseRequestSingle {
...
+    message AudioResponse {
+        bytes audio_data = 1;
+        AudioConfig audio_config = 2;
+        string text_data = 3;
+        bool end_of_response = 4;
+        oneof face_data_type {
+            VisemesData visemes_data = 5;
+            BlendShapesData blendshapes_data = 6;
+        }
+    }
```
Comment (manni901) [inline Proto/service/service.proto]: This is not the latest proto. Please get it from Yash.

File: Proto/service/service_pb2.py
Comment (manni901) [inline Proto/service/service_pb2.py:1]: Remove this file.

File: Proto/service/service_pb2_grpc.py
Comment (manni901) [inline Proto/service/service_pb2_grpc.py:1]: Remove this file.

**Review by manni901 — CHANGES_REQUESTED** [review]

File: Proto/service/service.proto (hunk 2 above)
Comment (AirSurfer09) [reply Proto/service/service.proto]: Will be updated with lipsync integration

File: Proto/service/service_pb2_grpc.py
Comment (AirSurfer09) [reply Proto/service/service_pb2_grpc.py:1]: Have deleted the file : https://github.com/Conv-AI/Convai-Web-SDK/pull/2/commits/4686caf10034b8e5a9d8146f6fe9f9680f74fb88

File: Proto/service/service_pb2.py
Comment (AirSurfer09) [reply Proto/service/service_pb2.py:1]: Have deleted the file : https://github.com/Conv-AI/Convai-Web-SDK/pull/2/commits/4686caf10034b8e5a9d8146f6fe9f9680f74fb88

File: .gitignore [hunk 3]
```diff
@@ -1,3 +1,5 @@
 node_modules
 /src
 /_proto
+Proto/service/service_pb2.py
+Proto/service/service_pb2_grpc.py
```
Comment (manni901) [inline .gitignore:5]: Remove this. These files should not be in the repository at all.

**Review by manni901 — APPROVED** [review]

**Review by crazylazylife — APPROVED** [review]

**Pull request merged**